docs/about/contributing/getting-started-developing.md

```diff
@@ -6,151 +6,357 @@ menu: menu
 toc:
     - title: Getting Started Developing
       url: "#getting-started-developing"
+    - title: Developing locally
+      url: "#developing-locally"
       active: true
-    - title: Running the Screwdriver API and UI
-      url: "#running-the-screwdriver-api-and-ui"
-    - title: Running SD-in-a-Box
-      url: "#running-sd-in-a-box"
+    - title: Developing locally with executor-queue
+      url: "#developing-locally-with-executor-queue-and-queue-service"
+    - title: Developing locally with custom launcher
+      url: "#developing-locally-with-a-custom-launcher-image"
 ---
 # Getting Started Developing
 
-There's three different ways to set up your dev environment locally in order to run and test Screwdriver locally:
+There's two different ways to set up your dev environment locally in order to run and test Screwdriver locally:
 
-1. **Run the Screwdriver API and UI locally** - ideal for testing out npm package dependencies that don't require a build to be run
-2. **Run SD-in-a-box** - ideal for running builds
-3. **Develop the Screwdriver API and UI locally** - ideal for developing npm package dependencies and testing simple builds locally.
+1. **Run the Screwdriver API and UI locally** - ideal for testing out npm package dependencies and run build locally. 
+See [Developing locally docs](#developing-locally).
+2. **Run SD-in-a-box** - ideal for running builds without manual configuration. See [Running SD-in-a-Box docs](../../cluster-management/running-locally).
 
-# Running the Screwdriver API and UI
-## Running the API
+## Developing Locally
 
-### Prerequisites
```

Comment:
would it still be good to have some sort of prereq section?

Reply:
added back